data/autoyast_sle15/autoyast_scc_up_ppc64le.xml

```diff
@@ -40,32 +41,11 @@
     <language>en_US</language>
     <languages>en_US</languages>
   </language>
-  <report>
-    <errors>
-      <log config:type="boolean">true</log>
-      <show config:type="boolean">true</show>
-      <timeout config:type="integer">0</timeout>
-    </errors>
-    <messages>
-      <log config:type="boolean">true</log>
-      <show config:type="boolean">true</show>
-      <timeout config:type="integer">0</timeout>
-    </messages>
-    <warnings>
-      <log config:type="boolean">true</log>
-      <show config:type="boolean">true</show>
-      <timeout config:type="integer">0</timeout>
-    </warnings>
-    <yesno_messages>
-      <log config:type="boolean">true</log>
-      <show config:type="boolean">true</show>
-      <timeout config:type="integer">0</timeout>
-    </yesno_messages>
-  </report>
   <partitioning config:type="list">
     <drive>
       <device>/dev/vda</device>
```

Comment:
@tinawang123 vda is hardcode one, can you please do some enhancement here?

Reply:
We can use a variable here if needed.

Reply:
Don't need. Our default autoyast profile also use this one. We'd better not add more settings at here.